docs/source/release_notes/release-notes.rst

```diff
@@ -63,11 +63,25 @@ develop
 
     *    - |userbreak| |fixed|
          - AtlasDB no longer embeds user-agents in metric names.
+           This affects both AtlasDB clients as well as TimeLock Server.
            All metrics are still available; however, metrics which previously included a user-agent component will no longer do so.
            For example, the timer ``com.palantir.timestamp.TimestampService.myUserAgent_version.getFreshTimestamp`` is now named ``com.palantir.timestamp.TimestampService.getFreshTimestamp``.
            This was necessary for compatibility with an internal log-ingestion tool.
            (`Pull Request <https://github.com/palantir/atlasdb/pull/2322>`__)
 
+    *    - |userbreak| |fixed|
+         - AtlasDB no longer embeds Cassandra host names in its metrics.
```

Comment:
Is there going to be any easy way to tell if certain Cassandra nodes are performing differently from the client's point of view?